src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Triggers/EventHubAsyncCollector.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.WebJobs.EventHubs
     /// Any user parameter that sends EventHub events will eventually get bound to this object. 
     /// This will queue events and send in batches, also keeping under the 256kb event hub limit per batch. 
     /// </summary>
-    internal class EventHubAsyncCollector : IAsyncCollector<EventData>
+    internal class EventHubAsyncCollector : IAsyncCollector<PartitionedValue>
```

Comment:
Isn't this a breaking change? Existing functions are bound to IAsyncCollector<EventData> and this is the implementation type?